get_initial_xero_tokens.ipynb

```diff
@@ -97,11 +103,11 @@
     {
      "data": {
       "text/plain": [
-       "['eyJhbGciOiJSUzI1NiIsImtpZCI6IjFDQUY4RTY2NzcyRDZEQzAyOEQ2NzI2RkQwMjYxNTgxNTcwRUZDMTkiLCJ0eXAiOiJKV1QiLCJ4NXQiOiJISy1PWm5jdGJjQW8xbkp2MENZVmdWY09fQmsifQ.eyJuYmYiOjE1ODUyMzU5NTEsImV4cCI6MTU4NTIzNzc1MSwiaXNzIjoiaHR0cHM6Ly9pZGVudGl0eS54ZXJvLmNvbSIsImF1ZCI6Imh0dHBzOi8vaWRlbnRpdHkueGVyby5jb20vcmVzb3VyY2VzIiwiY2xpZW50X2lkIjoiNUEyMjcxMTc2RUQxNDc2MkI5RkJGN0Y0RkNCMTE2RDYiLCJzdWIiOiJiMjEzODAxZjFhNTc1YTk3OTEyNGY1NWI5ZWQyNzdkZiIsImF1dGhfdGltZSI6MTU4NTIzNTY3OCwieGVyb191c2VyaWQiOiI5MDVlOWRmYS0zMjY2LTQ3MDAtYjFjMi1lN2RlMDU5ZmVlZTciLCJnbG9iYWxfc2Vzc2lvbl9pZCI6IjVkOWQxNDMzMDdhMDQ0OGE5ZGI2NzExMmFjMzliNzMzIiwianRpIjoiYjFhNjUxNWZlZWU0OWU1NjdjY2Y3ZThmNTMzYTQ4MjUiLCJzY29wZSI6WyJhY2NvdW50aW5nLnJlcG9ydHMucmVhZCIsImFjY291bnRpbmcuYXR0YWNobWVudHMucmVhZCIsInByb2plY3RzLnJlYWQiLCJhY2NvdW50aW5nLnNldHRpbmdzLnJlYWQiLCJmaWxlcy5yZWFkIiwiYWNjb3VudGluZy5qb3VybmFscy5yZWFkIiwiYWNjb3VudGluZy50cmFuc2FjdGlvbnMucmVhZCIsImFzc2V0cy5yZWFkIiwiYWNjb3VudGluZy5jb250YWN0cy5yZWFkIiwib2ZmbGluZV9hY2Nlc3MiXX0.pL9LXNB1gsQ3RbP6lklEKz3BU6IBEtHTlwb-kl-wkC44S7NLovzXkT3HkzYMNs0rqheTNx44yIxVl89kB43eyRrtxYar8zJgz-aUday9Qjz1bwcDE2L6w7-acHFEN84CQLNWBLynP5_FxPwRgW_CT5rwx2kxOVUcMgMhL4pzuvdNUmSF6OhDYlagSk_NdVCdFSt9VqvgU7C57kpg30S3ArQs3c_84t5nGZsMj2_tadlUL8t4zMqKqfwHYubnUKn7nuwgD4Ys5QvKrzlGGAtbq0GTR4dj0UMNMx3u6CvxJULRxO8bel4g1_ArcRMCZ_TO69nkk8DKBVpVUx7QT9SNLw',\n",
-       " '7c46814baeb05a9e8351f05a25291a0e351901ac9a35fcbb9e31dfe21f12c596']"
+       "['eyJhbGciOiJSUzI1NiIsImtpZCI6IjFDQUY4RTY2NzcyRDZEQzAyOEQ2NzI2RkQwMjYxNTgxNTcwRUZDMTkiLCJ0eXAiOiJKV1QiLCJ4NXQiOiJISy1PWm5jdGJjQW8xbkp2MENZVmdWY09fQmsifQ.eyJuYmYiOjE1ODgyNjI0MjQsImV4cCI6MTU4ODI2NDIyNCwiaXNzIjoiaHR0cHM6Ly9pZGVudGl0eS54ZXJvLmNvbSIsImF1ZCI6Imh0dHBzOi8vaWRlbnRpdHkueGVyby5jb20vcmVzb3VyY2VzIiwiY2xpZW50X2lkIjoiNUEyMjcxMTc2RUQxNDc2MkI5RkJGN0Y0RkNCMTE2RDYiLCJzdWIiOiJiMjEzODAxZjFhNTc1YTk3OTEyNGY1NWI5ZWQyNzdkZiIsImF1dGhfdGltZSI6MTU4ODI2MjQwNiwieGVyb191c2VyaWQiOiI5MDVlOWRmYS0zMjY2LTQ3MDAtYjFjMi1lN2RlMDU5ZmVlZTciLCJnbG9iYWxfc2Vzc2lvbl9pZCI6IjQ5Yjg3NDQzYTA3NzRhMzNiYmJiODhmMjdiMjk5ZjliIiwianRpIjoiNzEwZTcwMzAyNTJiNjk0OGE4NGRiNDA2NjZmZDE2YzEiLCJzY29wZSI6WyJhY2NvdW50aW5nLnJlcG9ydHMucmVhZCIsImFjY291bnRpbmcuYXR0YWNobWVudHMucmVhZCIsInByb2plY3RzLnJlYWQiLCJhY2NvdW50aW5nLnNldHRpbmdzLnJlYWQiLCJmaWxlcy5yZWFkIiwiYWNjb3VudGluZy5qb3VybmFscy5yZWFkIiwiYWNjb3VudGluZy50cmFuc2FjdGlvbnMucmVhZCIsImFzc2V0cy5yZWFkIiwiYWNjb3VudGluZy5jb250YWN0cy5yZWFkIiwib2ZmbGluZV9hY2Nlc3MiXX0.isg5wsmj4KcAFR55T1cReb7xdJp-hk5ud3F2BItP7FB4hHLMLY7wUN0s2EIv13m5wSu8v7E7DEHhvQvy7pK2bxdWdFf5cAvBJEXFvIp7-aMk_FmbiOx5Cw0zU_zw9QkM7Ltw94q2lAZCJDrIVSnpIH1tQeS9iknYecn5wgdzhpQ48mOvg63AYK_U4udngdBf1T9fyCYh_YnUCkJHIqUcPacv87E5qMgxHFILbfzaePHibnMV6m6NAanV_ZdKdAcHU92K_twS5V7z7vSKxJXUBlghNJIJa94qaIs8iyzhnROi1b51LiHurff9G0aP-uPhL8QCLcgtvUb8-Y2LVFWR1Q',\n",
+       " '950e9c71f302adedbbed6846bc7f8c1c1f4449aa2e633bcc597651c560cef752']"
```

Comment:
these tokens shouldn't be version controlled.
You can remove the returned values from `XeroFirstAuth()`:
```return [json_response['access_token'], json_response['refresh_token']]
```

Reply:
Ah good catch. Yeah these tokens are no longer valid now but good catch